docs/README.md

```diff
@@ -1,41 +1,25 @@
-# TypeScript Next.js example
+# Evergreen Docs
 
-This is a really simple project that shows the usage of Next.js with TypeScript.
+The following gives some information about getting the Evergreen docs site up and running locally.
 
-## Deploy your own
+# Spinning up the dev server
 
-Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+To get started locally, run the following commands:
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-typescript&project-name=with-typescript&repository-name=with-typescript)
-
-## How to use it?
-
-Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
-
-```bash
-npx create-next-app --example with-typescript with-typescript-app
-# or
-yarn create next-app --example with-typescript with-typescript-app
 ```
-
-Deploy it to the cloud with [Vercel](https://vercel.com/new?utm_source=github&utm_medium=readme&utm_campaign=next-example) ([Documentation](https://nextjs.org/docs/deployment)).
-
-## Notes
-
-This example shows how to integrate the TypeScript type system into Next.js. Since TypeScript is supported out of the box with Next.js, all we have to do is to install TypeScript.
-
-```
-npm install --save-dev typescript
+$ (docs): yarn install
+$ (docs): yarn dev
 ```
 
-To enable TypeScript's features, we install the type declarations for React and Node.
+Then, navigate to your browser and go to: `localhost:3000`.
 
-```
-npm install --save-dev @types/react @types/react-dom @types/node
-```
+# Adding a new documentation page
+
+Add an entry under the corresponding header in `IA.ts` - you can think of this as the "database" for all status relating to Evergreen components and documentation.
 
-When we run `next dev` the next time, Next.js will start looking for any `.ts` or `.tsx` files in our project and builds it. It even automatically creates a `tsconfig.json` file for our project with the recommended settings.
+From there, the directory structure in `docs/documentation/` contains the `.mdx` files which mirror the route structure for the docs site.
```

Comment:
Wonder if we also want to call out adding thumbnail images and add them to `public`

Reply:
Done! We can always come back here and just link to the `next.js` documentation. We're not doing anything fancy other than using the standard filesystem setup.